app/mailers/peer_to_peer_match_mailer.rb

```diff
@@ -2,15 +2,15 @@
 
 class PeerToPeerMatchMailer < ApplicationMailer
   def peer_to_peer_email(contribution, peer_alias:, message:)
-    peer_email_address = contribution.person.email
+    @peer_email_address = contribution.person.email
     email_subject = if contribution.ask?
                       "#{peer_alias} is offering to meet your ask"
                     elsif contribution.offer? # TODO: check if community resource when added
                       "#{peer_alias} would like to accept your offer"
                     end
 
-    mail(to: peer_email_address, subject: email_subject) do |format|
-      format.html { render locals: { message: message } }
+    mail(to: @peer_email_address, subject: email_subject) do |format|
+      format.html { render locals: { contribution: contribution, message: message } }
```

Comment:
I don't think i understand the distinction between using an instance var (`@peer_email_address`) vs. locals (`contribution`, `message`). The usage pattern in the template below looks pretty similar so wondering if we could consilidate on one or the other?